### PR TITLE
Add MDAnalysis/NanoVer notebook for analysis of NanoVer recordings

### PR DESCRIPTION
**Pull request by hjstroud**
This PR adds a notebook that details how to extract data from an iMD simulation recorded using NanoVer using MDAnalysis, and highlights some of the benefits of recording trajectories using the inbuilt NanoVer functionality.

As ever, comments and suggestions are welcome!

### CODE REVIEW

File: examples/mdanalysis/mdanalysis_nanover_recording.ipynb
Comment (rhoslynroebuck) [inline examples/mdanalysis/mdanalysis_nanover_recording.ipynb:1]: Looks great! 

The only issue I had was that the first line in the notebook about matplotlib widgets requires the `ipympl` package to be installed, but I can't see this in the requirements in the docs, and it didn't get installed for me as a dependency for any of the other packages. Would be good to add a note to the documentation (possibly also in the notebook?) that this needs to be installed.

**Review by rhoslynroebuck — COMMENTED** [review]
Looks great! See my comment on the notebook about dependencies

File: examples/mdanalysis/mdanalysis_nanover_recording.ipynb
Comment (hjstroud) [reply examples/mdanalysis/mdanalysis_nanover_recording.ipynb:1]: Ah interesting, this was something I found to be automatically installed in my environment, but it's good to know that it's not there automatically. I'll add something, thanks!

Comment (rhoslynroebuck) [reply examples/mdanalysis/mdanalysis_nanover_recording.ipynb:1]: Oh that's weird, maybe something about my env then? I'm running with jupyter notebook not jupyter lab, so could be something to do with that

**Review by rhoslynroebuck — APPROVED** [review]

**Comment by hjstroud** [comment]
Whilst making this notebook I noticed that the units of the system and user forces are not being converted properly from the NanoVer units (kJ/(mol.nm)) to the MDAnalysis units (kJ/(mol.A)). I have flagged this in Issue #227, and this will be fixed with PR #228.

**Review by rhoslynroebuck — APPROVED** [review]
Fab, all good.

**Pull request merged**